app/electron/main.ts

```diff
@@ -18,13 +18,42 @@ const defaultPort = 4466;
 
 function startServer(flags: string[] = []): ChildProcessWithoutNullStreams {
   const serverFilePath = path.join(process.resourcesPath, './electron/server');
-  return spawn(serverFilePath, [...flags], { shell: true });
+
+  const options = {shell: true, detached: false};
+  if (process.platform !== 'win32') {
+    // This makes the child processes a separate group, for easier killing.
+    options.detached = true;
+  }
+
+  return spawn(serverFilePath, [...flags], options);
 }
 
 let serverProcess: ChildProcessWithoutNullStreams | null;
 let intentionalQuit: boolean;
 let serverProcessQuit: boolean;
 
+function quitServerProcess() {
+  if (serverProcess) {
+    if (serverProcessQuit) {
+      log.error('server process already not running');
+    } else {
```

Comment:
Nitpick: I usually prefer an early return instead of a remaining else block (easier on the eyes IMO).
You can do the same for the first if block.

Reply:
Thanks. Fixed.